components/browser/search/src/test/java/mozilla/components/browser/search/SearchEngineManagerTest.kt

```diff
@@ -135,6 +139,7 @@ class SearchEngineManagerTest {
 
             shadow.sendBroadcast(Intent(Intent.ACTION_LOCALE_CHANGED))
 
+            countDownLatch.await(2, TimeUnit.SECONDS)
```

Comment:
Another option:

```Kotlin
launch(CommonPool) {}.join()
```

Reply:
Could be moved to an extension function too:

```Kotlin
    fun CoroutineDispatcher.waitForDone() {
        val dispatcher: CoroutineDispatcher = this
        runBlocking {
            launch(dispatcher) {}.join()
        }
    }


// ...

CommonPool.waitForDone()
```

Reply:
OK, I think launch(CommoPool) {}.join() is clearer here. Updated.